<?xml version="1.0"?>
<launch>
	<node pkg="map_server" type="map_server" name="map_server" args="$(find template_matching)/map_server/map2.yaml" />
	<node pkg="tf" type="static_transform_publisher" name="nav_origin_tf" args="24.0 0 0 0 0 0 /map /nav_origin 10" /> 
	<node pkg="template_matching"  type="tm" name="tm_demo" output="screen">
		<param name="distance_from_a" value="3.56" />
		<param name="distance_from_b" value="2.83" />
		<param name="distance_from_c" value="1.9" />
		<param name="map_path" value="$(find template_matching)/resources/map2.jpg" />
		<param name="template_A" value="$(find template_matching)/resources/tpl3.png" />
		<param name="template_B" value="$(find template_matching)/resources/tpl4.png" />
	</node>
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find template_matching)/demo.rviz" />
</launch>
